contracts/LinearPremiumPriceOracle.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity >=0.5.0;
+
+import "./SafeMath.sol";
+import "./StablePriceOracle.sol";
+
+contract LinearPremiumPriceOracle is StablePriceOracle {
+    using SafeMath for *;
+
+    uint GRACE_PERIOD = 90 days;
+
+    uint public initialPremium;
+    uint public premiumDecreaseRate;
+
+    constructor(DSValue _usdOracle, uint[] memory _rentPrices, uint _initialPremium, uint _premiumDecreaseRate) public
+        StablePriceOracle(_usdOracle, _rentPrices)
+    {
+        initialPremium = _initialPremium;
+        premiumDecreaseRate = _premiumDecreaseRate;
+    }
+
+    function _premium(string memory name, uint expires, uint duration) internal view returns(uint) {
```

Comment:
`uint duration` seems not used anywhere.

Reply:
No, but it's part of the interface.

Reply:
In `function price` at l57, you expressed unused variable with `uint /*expires*/`. Probably better to do it same so that the intention is clearer.